src/features/upload-file/upload-config.type.ts

```diff
@@ -0,0 +1,67 @@
+import { AWSOptions } from './adapters/aws-adapter'
+import { MimeType } from './mime-types.type'
+import BaseAdapter from './adapters/base-adapter'
+
+/**
+ * Configuration options for @admin-bro/upload feature
+ *
+ * @memberof module:@admin-bro/upload
+ */
+type UploadOptions = {
```

Comment:
I renamed this from UploadConfig to UploadOptions because all the configuration types are named Options in adminbro (AdminBroOptions, ResourceOptions etc)